src/libertem_live/detectors/merlin/acquisition.py

```diff
@@ -114,6 +122,7 @@ def initialize(self, executor):
 
         self._meta = DataSetMeta(
             shape=Shape(self._nav_shape + sig_shape, sig_dims=2),
+            array_backends=(CUPY, CUDA, NUMPY),
```

Comment:
Same in the Dectris case, could this be the `@property` ?


```suggestion
            array_backends=self.array_backends,
```